instrumentation-security/spring-webmvc-3.1.0/src/main/java/org/springframework/web/servlet/handler310/SpringHelper.java

```diff
@@ -0,0 +1,26 @@
+package org.springframework.web.servlet.handler310;
+
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.ApplicationURLMapping;
+import com.newrelic.api.agent.security.instrumentation.helpers.*;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.condition.PatternsRequestCondition;
+import org.springframework.web.servlet.mvc.method.RequestMappingInfo;
+
+import java.util.Iterator;
+
+public class SpringHelper {
+    public static <T> void gatherURLMappings(T mapping){
+        try {
+            RequestMappingInfo mappingInfo = (RequestMappingInfo) mapping;
+            for (RequestMethod method : mappingInfo.getMethodsCondition().getMethods()) {
+                PatternsRequestCondition patternsCondition = mappingInfo.getPatternsCondition();
+                if (patternsCondition != null)
```

Comment:
This pattern check should be even before L:16 and please always add curly braces after if and for even if the code is one liner.

Reply:
Fixed this in commit [c548fde](https://github.com/newrelic/csec-java-agent/pull/53/commits/c548fde01b1e9223a5d86171749fedcf35f7a406)